src/app/tests/integration/README.md

```diff
@@ -0,0 +1,44 @@
+# CHIP Example IM Application Tutorial
+
+## Introduction
+
+The CHIP IM example application shows you how to implement a CHIP application
+program using IM protocols.
+
+CHIP Protocols are, essentially, implementations of specific protocols over the
+CHIP transport. Furthermore, when two CHIP nodes are exchanging messages of a
+particular CHIP protocol, they do so over a construct called a CHIP Exchange
+which is a description of a CHIP-based conversation over a CHIP protocol. A CHIP
+Exchange is characterised by the ExchangeContext object, and every CHIP node
+must create an ExchangeContext object before initiating a CHIP conversation.
+
+After constructing a CHIP ExchangeContext, CHIP messages are sent and received
+using the ChipMessageLayer class which sends the CHIP message over a chosen
+transport (TCP, UDP, or CRMP).
+
+## Building
```

Comment:
Suggest just removing this section. It doesn't match the general build documentation and it makes assumptions like use of "clang".

Reply:
Done, thanks